server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader9to10.java

```diff
@@ -111,6 +111,31 @@
  * {@link #dropSortedMapWALFiles(ServerContext)}. For more information see the following issues:
  * <a href="https://github.com/apache/accumulo/issues/2117">#2117</a> and
  * <a href="https://github.com/apache/accumulo/issues/2179">#2179</a>
+ *
+ * The following methods were created for External compactions:
+ * {@link #createExternalCompactionNodes(ServerContext)}, {@link #setMetaTableProps(ServerContext)}
+ *
+ * Improvements to the metadata and root tables were made in this version. See pull request
+ * <a href="https://github.com/apache/accumulo/pull/1174">#1174</a> for more details. The
```

Comment:
If the details don't matter for understanding the current code, we probably don't need this here at all. The git history should suffice.